public/js/NeighbourImages.js

```diff
@@ -54,18 +73,54 @@ NeighbourImages.prototype.getImages = function(query_id) {
 /**
  * setImages: Convert base64 encoded strings to images and set them in the DOM.
  *
+ * Clicking on 'neighbour' images sets the corresponding point in the PCA plot as the
+ * active point.
+ *
  * @this {NeighbourImages}
  */
 NeighbourImages.prototype.setImages = function() {
     var self = this;
+    var $nebula0 = $('#nebula-0');
+
+    // make all img frames empty
+    $nebula0
+        .attr('src', '//:0')
+        .attr('title', 'not found');
+
+    $('.thumbnail > img')
```

Comment:
This syntax is really throwing me off. Can you point me to a writeup on it? (Either on our repo or in the wider web, linked from our repo.)